src/Core/SonarLintNotifications/NotificationSetting.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.Core.SonarLintNotifications
+{
+    public interface INotificationSetting
+    {
+        string Key { get; }
```

Comment:
Is the Key different from NotificationId in the notification object?

Reply:
Yeah, it would make sense to have them same